sparkly/writer.py

```diff
@@ -255,6 +275,173 @@ def write_partition_to_kafka(messages):
 
         rdd.mapPartitions(write_partition_to_kafka).count()
 
+    def redis(self,
+              key_by,
+              key_prefix=None,
+              key_delimiter='.',
+              group_by_key=False,
+              expire=None,
+              compression=None,
+              max_pipeline_size=100,
+              parallelism=None,
+              mode='overwrite',
+              host=None,
+              port=6379,
+              db=0,
+              redis_client_init=None):
+        """Write a dataframe to Redis as JSON.
+
+        Args:
+            key_by (list[str]): Column names that form the redis key for
+                each row. The columns are concatenated in the order they
+                appear.
+            key_prefix (str|None): Common prefix to add to all keys from
+                this DataFrame. Useful to namespace DataFrame exports.
+            key_delimiter (str|.): Characters to delimit different columns
+                while forming the key.
+            group_by_key (bool|False): If set, group rows that share the
+                same redis key together in an array before exporting. By
+                default if multiple rows share the same redis key, one
+                will overwrite the other.
+            expire (int|None): Expire the keys after this number of seconds.
```

Comment:
The official redis client refers to this as `ex`(pire) in their `set(...)` function linked above. If you want to pass a timestamp there is an `ExpireAt` command in the official documentation, which would correspond to an `expireAt` URL parameter. I think this is a pretty straightforward association with Redis' official manuals, so I don't think `expire` is ambiguous at all. `TTL` provides read access to whatever the expire flag is and as such I think is more confusing to use in this case.